third_party/nvidia/hopper/include/Transforms/Passes.td

```diff
@@ -102,4 +102,12 @@ def NVGPUTestPingPongSync : Pass<"nvgpu-test-ping-pong-sync", "mlir::ModuleOp">
   ];
 }
 
+def NVGPUTest1DTMEMAlloc : Pass<"nvgpu-test-1D-tmem-alloc", "mlir::ModuleOp"> {
+  let summary = "test allocating tmem for a 1D tensor that should be passed across partitions.";
+
+  let description = "This pass rewrites IR annotated with a tmem.start and tmem.end into a load and store thorugh tmem";
```

Comment:
I don't seem to see `tmem.end` used anywhere. Is it still supported?

Reply:
This is out of date from before I realized I could just check when the partitions didn't match. I'll update this code.